pkg/providers/aws/provider_postgres.go

```diff
@@ -696,11 +696,6 @@ func verifyPendingModification(mi *rds.ModifyDBInstanceInput, pm *rds.PendingMod
 			pendingModifications = false
 		}
 	}
-	if mi.AllocatedStorage != nil && pm.AllocatedStorage != nil {
-		if *mi.AllocatedStorage == *pm.AllocatedStorage {
-			pendingModifications = false
-		}
-	}
```

Comment:
Should we be replacing this with MaxAllocatedStorage also?

Reply:
No, this is to check for pending modifications, such that we do not continue calling modify instance until the next maintenance window hits. Id the modification is in a pending state. 

max allocation storage is modified instantly

Reply:
Ok, I see that documented here https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.DBInstance.Modifying.html under `Settings for your DB Instances`